tapfreighter/wallet.go

```diff
@@ -67,14 +67,15 @@ type Wallet interface {
 	// data which assists in processing the virtual transaction: passive
 	// asset re-anchors and the Taproot Asset level commitment of the
 	// selected assets.
-	FundAddressSend(ctx context.Context,
+	FundAddressSend(ctx context.Context, bip86ScriptKeysOnly bool,
 		receiverAddrs ...*address.Tap) (*FundedVPacket, error)
 
 	// FundPacket funds a virtual transaction, selecting assets to spend
 	// in order to pay the given recipient. The selected input is then added
 	// to the given virtual transaction.
 	FundPacket(ctx context.Context, fundDesc *tapsend.FundingDescriptor,
-		vPkt *tappsbt.VPacket) (*FundedVPacket, error)
+		vPkt *tappsbt.VPacket,
+		bip86ScriptKeysOnly bool) (*FundedVPacket, error)
```

Comment:
With the comment above, the new enum can possibly go in the `tapsend.FundingDescriptor`.